promtimer/promtimer.py

```diff
@@ -261,14 +268,19 @@ def main():
         max_time = datetime.datetime.fromtimestamp(times[1]).isoformat()
         refresh = ''
     else:
-        stats_sources = cbstats.ServerNode.get_stats_sources(args.cluster,
-                                                             args.user,
-                                                             args.password)
+        if args.nodes:
```

Comment:
elif?

Reply:
The nested if is necessary because there's some code we want to run in both cases